src/main/scala/io/iohk/ethereum/crypto/ECIESPublicKeyEncoder.scala

```diff
@@ -0,0 +1,13 @@
+package io.iohk.ethereum.crypto
+
+import org.spongycastle.crypto.KeyEncoder
+import org.spongycastle.crypto.params.AsymmetricKeyParameter
+import org.spongycastle.crypto.params.ECPublicKeyParameters
+
+/**
+  * Created by Anton Nashatyrev on 01.10.2015.
```

Comment:
Probably author of this class in EthereumJ, since I ported it 1:1 I decided that it should stay, as it turns out this class is simple and can be "inlined"